client/app/components/admin/StatusBlock.jsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/prop-types */
+
+import { isString, toPairs } from 'lodash';
+import React from 'react';
+
+import List from 'antd/lib/list';
+import Card from 'antd/lib/card';
+import Tooltip from 'antd/lib/tooltip';
+import { TimeAgo } from '@/components/TimeAgo';
+
+import { toHuman } from '@/filters';
+
+export function General({ info }) {
+  info = toPairs(info);
+  return (
+    <Card title="General" size="small">
+      {(info.length === 0) && (
+        <div className="text-muted text-center">No data</div>
+      )}
+      {(info.length > 0) && (
+        <List
+          size="small"
+          itemLayout="vertical"
+          dataSource={info}
+          renderItem={([name, value]) => (
+            <List.Item extra={<span className="badge">{value}</span>}>
+              {toHuman(name)}
+            </List.Item>
+          )}
+        />
+      )}
+    </Card>
+  );
+}
+
+export function DatabaseMetrics({ info }) {
+  return (
+    <Card title="Redash Database" size="small">
+      {(info.length === 0) && (
+        <div className="text-muted text-center">No data</div>
+      )}
+      {(info.length > 0) && (
+        <List
+          size="small"
+          itemLayout="vertical"
+          dataSource={info}
+          renderItem={([name, size]) => (
+            <List.Item extra={<span className="badge">{size}</span>}>
+              {name}
+            </List.Item>
+          )}
+        />
+      )}
+    </Card>
+  );
+}
+
+export function Queues({ info }) {
+  info = toPairs(info);
+  return (
+    <Card title="Queues" size="small">
+      {(info.length === 0) && (
+        <div className="text-muted text-center">No data</div>
+      )}
+      {(info.length > 0) && (
+        <List
+          size="small"
+          itemLayout="vertical"
+          dataSource={info}
+          renderItem={([name, queue]) => (
+            <List.Item extra={<span className="badge">{queue.size}</span>}>
+              {name}
+              <Tooltip
+                className="m-l-5"
+                title={isString(queue.data_sources) && (queue.data_sources !== '') ? queue.data_sources : 'n/a'}
+                mouseLeaveDelay={0}
+              >
```

Comment:
The Tooltip is not needed -- we no longer send the data sources list.

Reply:
On preview instance it works :thinking_face:

Reply:
Because it doesn't run the latest version of the code :) Try locally.